static/css/preloader.css

```diff
@@ -1,13 +1,22 @@
 /* preloader */
+
 .folding-cube ul{
-    position: fixed;
-    top: 48vh;
-    left: 45vw;
+    position: absolute;
```

Comment:
make the position fixed else if we scroll it will move.

Reply:
done!